src/unreachable.ts

```diff
@@ -0,0 +1,6 @@
+// this functions signifies code that is never executed. Typescript should
```

Comment:
```suggestion
// this functions signifies code that is never executed. TypeScript should
```

Reply:
Damn. Bad copy+pasta, but I really should have checked 🤦